sm6115.c

```diff
@@ -81,131 +97,131 @@ static struct debug_mux gpu_cc = {
-	{ "gcc_qupv3_wrap_0_s_ahb_clk", &gcc, 0x68 },
-	{ "gcc_sdcc1_ahb_clk", &gcc, 0xf3 },
-	{ "gcc_sdcc1_apps_clk", &gcc, 0xf2 },
-	{ "gcc_sdcc1_ice_core_clk", &gcc, 0xf4 },
-	{ "gcc_sdcc2_ahb_clk", &gcc, 0x66 },
-	{ "gcc_sdcc2_apps_clk", &gcc, 0x65 },
-	{ "gcc_sys_noc_cpuss_ahb_clk", &gcc, 0x9 },
-	{ "gcc_sys_noc_ufs_phy_axi_clk", &gcc, 0x19 },
-	{ "gcc_sys_noc_usb3_prim_axi_clk", &gcc, 0x18 },
-	{ "gcc_ufs_phy_ahb_clk", &gcc, 0x111 },
-	{ "gcc_ufs_phy_axi_clk", &gcc, 0x110 },
-	{ "gcc_ufs_phy_ice_core_clk", &gcc, 0x117 },
-	{ "gcc_ufs_phy_phy_aux_clk", &gcc, 0x118 },
-	{ "gcc_ufs_phy_rx_symbol_0_clk", &gcc, 0x113 },
-	{ "gcc_ufs_phy_tx_symbol_0_clk", &gcc, 0x112 },
-	{ "gcc_ufs_phy_unipro_core_clk", &gcc, 0x116 },
-	{ "gcc_usb30_prim_master_clk", &gcc, 0x5c },
-	{ "gcc_usb30_prim_mock_utmi_clk", &gcc, 0x5e },
-	{ "gcc_usb30_prim_sleep_clk", &gcc, 0x5d },
-	{ "gcc_usb3_prim_phy_com_aux_clk", &gcc, 0x5f },
-	{ "gcc_usb3_prim_phy_pipe_clk", &gcc, 0x60 },
-	{ "gcc_vcodec0_axi_clk", &gcc, 0x142 },
-	{ "gcc_venus_ahb_clk", &gcc, 0x143 },
-	{ "gcc_venus_ctl_axi_clk", &gcc, 0x141 },
-	{ "gcc_video_ahb_clk", &gcc, 0x36 },
-	{ "gcc_video_axi0_clk", &gcc, 0x3c },
-	{ "gcc_video_throttle_core_clk", &gcc, 0x4a },
-	{ "gcc_video_vcodec0_sys_clk", &gcc, 0x13f },
-	{ "gcc_video_venus_ctl_clk", &gcc, 0x13d },
-	{ "gcc_video_xo_clk", &gcc, 0x3e },
-	//{ "gpu_cc_debug_mux", &gcc, 0xe7 },
-	//{ "mc_cc_debug_mux", &gcc, 0x9e },
-	{ "measure_only_cnoc_clk", &gcc, 0x1a },
-	{ "measure_only_ipa_2x_clk", &gcc, 0xc6 },
-	{ "measure_only_snoc_clk", &gcc, 0x7 },
-	{ "disp_cc_mdss_ahb_clk", &gcc, 0x42, &disp_cc, 0x1a },
-	{ "disp_cc_mdss_byte0_clk", &gcc, 0x42, &disp_cc, 0x12 },
-	{ "disp_cc_mdss_byte0_intf_clk", &gcc, 0x42, &disp_cc, 0x13 },
-	{ "disp_cc_mdss_esc0_clk", &gcc, 0x42, &disp_cc, 0x14 },
-	{ "disp_cc_mdss_mdp_clk", &gcc, 0x42, &disp_cc, 0xe },
-	{ "disp_cc_mdss_mdp_lut_clk", &gcc, 0x42, &disp_cc, 0x10 },
-	{ "disp_cc_mdss_non_gdsc_ahb_clk", &gcc, 0x42, &disp_cc, 0x1b },
-	{ "disp_cc_mdss_pclk0_clk", &gcc, 0x42, &disp_cc, 0xd },
-	{ "disp_cc_mdss_rot_clk", &gcc, 0x42, &disp_cc, 0xf },
-	{ "disp_cc_mdss_vsync_clk", &gcc, 0x42, &disp_cc, 0x11 },
-	{ "disp_cc_sleep_clk", &gcc, 0x42, &disp_cc, 0x24 },
-	{ "disp_cc_xo_clk", &gcc, 0x42, &disp_cc, 0x23 },
-	{ "gpu_cc_ahb_clk", &gcc, 0xe7, &gpu_cc, 0x10 },
-	{ "gpu_cc_crc_ahb_clk", &gcc, 0xe7, &gpu_cc, 0x11 },
-	{ "gpu_cc_cx_gfx3d_clk", &gcc, 0xe7, &gpu_cc, 0x1a },
-	{ "gpu_cc_cx_gmu_clk", &gcc, 0xe7, &gpu_cc, 0x18 },
-	{ "gpu_cc_cx_snoc_dvm_clk", &gcc, 0xe7, &gpu_cc, 0x15 },
-	{ "gpu_cc_cxo_aon_clk", &gcc, 0xe7, &gpu_cc, 0xa },
-	{ "gpu_cc_cxo_clk", &gcc, 0xe7, &gpu_cc, 0x19 },
-	{ "gpu_cc_gx_cxo_clk", &gcc, 0xe7, &gpu_cc, 0xe },
-	{ "gpu_cc_gx_gfx3d_clk", &gcc, 0xe7, &gpu_cc, 0xb },
-	{ "gpu_cc_sleep_clk", &gcc, 0xe7, &gpu_cc, 0x16 },
+	{ "perfcl_clk", &cpu_cc, 0x1 },
+	{ "pwrcl_clk", &cpu_cc, 0x0 },
+	//{ "cpu_cc_debug_mux", &gcc.mux, 0xaf },
+	//{ "disp_cc_debug_mux", &gcc.mux, 0x42 },
```

Comment:
IMHO these comments are even more useless now, can we just remove them?

Reply:
Yep, why not.

Reply:
Dropped most of useless comments, retained only those debug muxes that we do not implement for now.